utils/build.ps1

```diff
@@ -1167,6 +1167,38 @@ function Get-SwiftSDK {
   return ([IO.Path]::Combine((Get-PlatformRoot $OS), "Developer", "SDKs", "$Identifier.sdk"))
 }
 
+function Get-CMakePath {
+  try {
+    $cmake_bin = (Get-Command "cmake.exe" -ErrorAction Stop).Source
+  } catch {
+      $vswhere = "${env:ProgramFiles(x86)}\Microsoft Visual Studio\Installer\vswhere.exe"
+      $VSInstallPath = & $vswhere -nologo -latest -products * -property installationPath
+      if (Test-Path "${VSInstallPath}\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin") {
```

Comment:
You should add in a `-Type Container` to ensure that it is a directory.

Reply:
`-PathType Container`. Done.